src/aws/ffmpeg-transmux-live-image/entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+echo "Transmuxing video ${OUPUT_MP4_FILENAME}"
+ffmpeg -v error -i "${HLS_MANIFEST_ENDPOINT}" -codec copy -f mp4 "${OUPUT_MP4_FILENAME}"
+echo "Generating thumbnail ${OUTPUT_THUMBNAIL_FILENAME}"
+ffmpeg -v error -ss 00:00:01.000 -i "${HLS_MANIFEST_ENDPOINT}" -vframes 1 "${OUTPUT_THUMBNAIL_FILENAME}"
+echo "Copying ${OUPUT_MP4_FILENAME} to s3"
+aws s3 cp "./${OUPUT_MP4_FILENAME}" "s3://${DESTINATION_BUCKET_NAME}/${VIDEO_BUCKET_KEY}" --region "${DESTINATION_BUCKET_REGION:-"eu-west-1"}"
+echo "Copying ${OUTPUT_THUMBNAIL_FILENAME} to s3"
+aws s3 cp "./${OUTPUT_THUMBNAIL_FILENAME}" "s3://${DESTINATION_BUCKET_NAME}/${THUMBNAIL_BUCKET_KEY}" --region "${DESTINATION_BUCKET_REGION:-"eu-west-1"}"
```

Comment:
To facilitate the reading of this file and to avoid repetition, I suggest to declare `DESTINATION_BUCKET_REGION` on top of this script to handle the default value : 

```shell
declare -r DESTINATION_BUCKET_REGION="${DESTINATION_BUCKET_REGION:-"eu-west-1"}"
```

and then : 

```suggestion
aws s3 cp "./${OUPUT_MP4_FILENAME}" "s3://${DESTINATION_BUCKET_NAME}/${VIDEO_BUCKET_KEY}" --region "${DESTINATION_BUCKET_REGION}"
echo "Copying ${OUTPUT_THUMBNAIL_FILENAME} to s3"
aws s3 cp "./${OUTPUT_THUMBNAIL_FILENAME}" "s3://${DESTINATION_BUCKET_NAME}/${THUMBNAIL_BUCKET_KEY}" --region "${DESTINATION_BUCKET_REGION}"
```